### PR TITLE
86due3k54: Display monthly sale and other important data per moth

### PR DESCRIPTION
**Pull request by lualreye**

### CODE REVIEW

File: src/components/Graphs/SalesGraph.vue [hunk 1]
```diff
@@ -115,5 +115,6 @@ onMounted(() => {
 <style lang="scss" scoped>
 #salesChart {
   height: 320px;
+  width: 100%
```
Comment (yeyodev1) [inline src/components/Graphs/SalesGraph.vue:118]: good catch

File: src/interfaces/sales.interface.ts [hunk 2]
```diff
@@ -6,4 +6,10 @@ export interface SalesDataItem {
 
 export interface SalesOrders {
   annualSalesByMonth: SalesDataItem[]
+}
```
Comment (yeyodev1) [inline src/interfaces/sales.interface.ts:9]: add a ";"

File: src/interfaces/sales.interface.ts [hunk 3]
```diff
@@ -6,4 +6,10 @@ export interface SalesDataItem {
...
+
+export interface currentMonthDetails {
+  currentMonthRevenue: number;
+  ordersOpen: number;
+  salesMonthClosed: number;
 }
```
Comment (yeyodev1) [inline src/interfaces/sales.interface.ts:15]: same here

File: src/views/app/Dashboard/Index.vue [hunk 4]
```diff
@@ -1,38 +1,73 @@
...
-import { watch } from 'vue';
+import { computed, watch } from 'vue';
 
 import useSalesStore from '@/store/sales';
 import useRestaurantStore from '@/store/restaurant';
-import SalesGraph from '@/components/Graphs/SalesGraph.vue';
 import GlobalLoading from '@/components/GlobalLoading.vue';
+import SalesGraph from '@/components/Graphs/SalesGraph.vue';
+import DetailCard from '@/views/app/Dashboard/components/DetailCard.vue';
+import { formatPriceToDisplay } from '@/utils/inputFormats';
```
Comment (yeyodev1) [inline src/views/app/Dashboard/Index.vue:8]: this is too long to be the last one, check the long of the imports and order it

File: src/views/app/Dashboard/Index.vue [hunk 5]
```diff
@@ -1,38 +1,73 @@
...
+const hasDetailData = computed(() => salesStore.currentMonthRevenue && salesStore.ordersOpen && salesStore.salesMonthClosed)
+const currentRevenue = computed(() => formatPriceToDisplay(String(salesStore.currentMonthRevenue)));
+
+async function getDashboardData(restaurantId: string): Promise<void> {
   if (!salesStore.salesPerMonth) {
     await salesStore.getSalesPerMonth(restaurantId);
   }
+  if (!hasDetailData.value) {
+    await salesStore.getSalesCurrentMonth(restaurantId);
+  }
```
Comment (yeyodev1) [inline src/views/app/Dashboard/Index.vue:23]: add a ";"

File: src/views/app/Dashboard/Index.vue [hunk 6]
```diff
@@ -1,38 +1,73 @@
...
 }
```
Comment (yeyodev1) [inline src/views/app/Dashboard/Index.vue:24]: same here

File: src/views/app/DeliveryView/components/Card/PickerCard.vue [hunk 7]
```diff
@@ -16,7 +16,7 @@ const props = defineProps({
 
 const messageCard = computed(() => {
   return props.pickerFloatIsActive
-  ? 'Estas trabajando Picker'
+  ? 'Estas usando Picker'
```
Comment (yeyodev1) [inline src/views/app/DeliveryView/components/Card/PickerCard.vue:19]: goood catch

**Review by yeyodev1 — CHANGES_REQUESTED** [review]

File: src/views/app/Dashboard/Index.vue (hunk 5 above)
Comment (lualreye) [reply src/views/app/Dashboard/Index.vue:23]: a block in javascript does not use semicolon

File: src/interfaces/sales.interface.ts (hunk 3 above)
Comment (lualreye) [reply src/interfaces/sales.interface.ts:15]: a block in javascript does not use semicolon

File: src/interfaces/sales.interface.ts (hunk 2 above)
Comment (lualreye) [reply src/interfaces/sales.interface.ts:9]: done

File: src/views/app/Dashboard/Index.vue (hunk 5 above)
Comment (yeyodev1) [reply src/views/app/Dashboard/Index.vue:23]: ñiñiñi

**Review by yeyodev1 — APPROVED** [review]
looks awesome

**Pull request merged**